.github/workflows/tests_gpu_python.yml

```diff
@@ -93,19 +81,16 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-tags: true
-          path: main
 
       - name: Switch to release build of Lightning
-        if: inputs.lightning-version == 'release'
+        if: inputs.lightning-version == 'release' && matrix.pl_backend == 'lightning.qubit'
```

Comment:
why do we need to check pl_backend? Have you tested these workflows in PL-Lightning compatible CI workflows?

Reply:
I didn't test. I think we can improve on this later. For now, I'm mimicking the behaviour we had before.